.github/workflows/test.yml

```diff
@@ -0,0 +1,55 @@
+name: xpra
+
+on: [push, pull_request]
+
+jobs:
+  unittest:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: "apt-get install"
+      run: |
+        # see docs/Build/Debian.md
+        sudo apt-get update
+        sudo apt-get -y install libx11-dev libxtst-dev libxcomposite-dev libxdamage-dev libxres-dev \
+                libxkbfile-dev python-all-dev pandoc libsystemd-dev \
+                libgtk-3-dev python3-dev python3-cairo-dev python-gi-dev cython3 \
+                xauth x11-xkb-utils \
+                xvfb python3-cairo python3-gi-cairo \
+                python3-opengl python3-lz4 python3-rencode python3-pil \
+                libturbojpeg-dev libwebp-dev python3-pil \
+                libx264-dev libvpx-dev yasm \
+                python3-numpy \
```

Comment:
Why is this one here? I thought it should only be needed with the `nvidia` codecs - which are not available when running the unit tests.

Reply:
I started by concatenating all of the build dependency lists in `docs/Build/Debian.md`, then adding some missing pieces (eg. `liblz4-dev`.).  I didn't remove any.  I think it would be nice to prune down to a minimal dependency list.  When I encounter a new OSS project, I tend to look at CI configuration for "as built" documentation of dependencies.

However, I suspect this would need more time than I have at present to try possible combinations and/or learn about the dependency chains.  (I see a bewildering number of possible combinations...)

Reply:
Removed yasm: 3c76c3254c4ae402c2b1063714a0e7fd91d9be1b

Reply:
A lot of these dependencies aren't needed: c4dca10b3c8aab74442fb3b5fcd7353f8288ad5d
The only test that uses numpy is:
https://github.com/Xpra-org/xpra/blob/c4dca10b3c8aab74442fb3b5fcd7353f8288ad5d/tests/unittests/unit/server/motion_test.py#L108-L113
And it will now just skip this one test case.
(numpy is heavy - don't install it if we don't have to)